src/components/formatters/UsdFormat/index.tsx

```diff
@@ -19,6 +20,7 @@ function UsdFormat({ value, style, decimalScale = 2 }: Props) {
       fixedDecimalScale
       decimalScale={decimalScale}
       prefix="$"
+      suffix={showSuffix ? ' USD' : undefined}
```

Comment:
send the suffix as prop, so in the future we can reuse this component with other currencies

`suffix={suffix}`